WORKSPACE

```diff
@@ -121,6 +121,14 @@ new_http_archive(
 
 DEBUG_BASE_IMAGE_SHA = "3f57ae2aceef79e4000fb07ec850bbf4bce811e6f81dc8cfd970e16cdf33e622"
 
+new_http_archive(
+    name = "docker_centos",
+    build_file = "BUILD.centos",
+    sha256 = "9e945e0ea9bf66c73e6695f7d70a9c0bbd3f8f5c67b9a44e3127a4ac495e2d85",
+    type = "xz",
+    url = "https://github.com/CentOS/sig-cloud-instance-images/raw/16dab97b0ce72b1db7a2f9b02c76e452cb0a63cb/docker/centos-7-docker.tar.xz",
```

Comment:
A sha like `16dab97b0ce72b1db7a2f9b02c76e452cb0a63cb` should have an explanation. 
Does this correspond to a tag? If not what is the date that it was published.